nodeList.json

```diff
@@ -8,20 +8,20 @@
       "floor":"0",
       "meshHardware":"Unknown",
       "IPV6Address":"Unknown",
-      "lastPing":"-1",
-      "rtt":"0"
+      "status":"active",
+      "dateAdded":"2017-18-15T00:00:00.0Z"
    },
    {
       "name":"Willowdale Mesh Node",
```

Comment:
I wasn't aware that this had been added prior to confirming. Can we identify what nodes in the list are like that?
I'd prefer they not be added in unless confirmed/added by the individual